db/warehouse/migrate/20240320190835_add_service_referral_columns_to_report_path_client.rb

```diff
@@ -0,0 +1,6 @@
+class AddServiceReferralColumnsToReportPathClient < ActiveRecord::Migration[6.1]
+  def change
+    add_column :hud_report_path_clients, :cmh_service_provided, :boolean
+    add_column :hud_report_path_clients, :cmh_referral_provided_and_attained, :boolean
```

Comment:
these should probably be:
```suggestion
    add_column :hud_report_path_clients, :cmh_service_provided, :boolean, default: false, null: false
    add_column :hud_report_path_clients, :cmh_referral_provided_and_attained, :boolean, default: false, null: false
```
Though I doubt that will have any functional difference.